docs/decisions/0010-multiple-event-types-per-topic.rst

```diff
@@ -0,0 +1,34 @@
+10. Multiple event types per topic
+##################################
+
+Status
+******
+
+- **Accepted** DATE TO BE ADDED BEFORE MERGING
```

Comment:
Nit: You can drop the `-` at the beginning.

This is super minor, but I just created https://github.com/openedx/open-edx-proposals/pull/475 to help avoid this in the future, if you want to give it a quick review.